### DISCUSSION TITLE
What's the difference between experimental-edge vs edge?

### DISCUSSION THREAD

**Discussion by shinebayar-g** [vercel/next.js#44871] [Help] [answered]
https://nextjs.org/docs/api-routes/edge-api-routes#basic doc says use:
```ts
export const config = {
  runtime: 'edge',
}
```

But there is another version mentioned here https://nextjs.org/docs/advanced-features/react-18/switchable-runtime#runtime-differences

> `edge` or `experimental-edge` if using Next.js Rendering

Is there any difference between these 2?

**Comment by icyJoseph** [comment, marked as answer]
As the Edge feature approaches maturity, the runtime name changes from `experimental-edge` to `edge`.

If I understand correctly, for "rendering" using Next.js, you still have to use `experimental-edge`, because that part of the edge feature is not mature enough, just yet.

API routes usage of the edge is mature, so we ought to use the `edge` runtime, not because it is fundamentally different, but because they are "done" implementing API routes runtime on the edge.
